app/workers/gamedev_worker.rb

```diff
@@ -0,0 +1,23 @@
+class GamedevWorker
+  include Sidekiq::Worker
+  def perform
+    response = RubyStackoverflow.questions(
```

Comment:
When you have many lines within a method, it is better to create additional methods to simplify the code and make it easier to read. For example:

```ruby
class GamedevWorker
  include Sidekiq::Worker
  def perform
    fetch_questions.each do |question|
      create_post(question)
    end
  end

  def fetch_questions
    response = RubyStackoverflow.questions(
      order: "asc",
      filter: "withbody",
      tagged: "ruby",
      pagesize: 100)
    response.data
  end

  def create_post(question)
    ExternalPost.create(
      title: question.title,
      user_display_name: question.owner[:display_name],
      body_html: question.body,
      source_url: question.link,
      external_id: question.question_id)
  end
# ...
```